scripts/quick-deploy-service.ts

```diff
@@ -326,6 +336,12 @@ async function main(): Promise<void> {
           { title: 'Entitlement Service', value: 'entitlement-service' },
         ],
       },
+      {
+        type: 'text',
+        name: 'dockerUsername',
+        message: 'Enter the Docker username',
+        initial: username,
```

Comment:
if the username is not supplied, can we handle that scenario bit better?

either make `prompt` not proceed with "empty" values or we fail early with an error - rather than proceeding upto build step and then only giving some error :)